packages/teams-js/src/public/media.ts

```diff
@@ -412,30 +410,38 @@
      * @param mediaEvent indicates what the event that needs to be signaled to the host client
      * Optional; @param callback is used to send app if host client has successfully handled the notification event or not
      */
-    protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: SdkError) => void): void {
```

Comment:
>callback(isSupported) [](http://example.com/codeflow?start=17&length=21)

What does it mean to return a boolean to this callback? The reason the callback existed was to pass the error back to the caller I think. Passing a false here won't do much (and won't ever get hit because isMobileApiSupported throws in the false case now).

Reply:
I think in order to maintain backcompatibility, we still want the callback to take in SdkErrors if the callback is present. In this function, if the callback is specified we would need to catch the exception from isMobileApiSupported and then pass it to the callback. If there was no callback, instead we just let isMobileApiSupported's exception keep bubbling up

Reply:
(Alternatively you could keep this function the way it is but with no callback at all, and then do the catch and pass to callback if callback is defined in the stop function. That might be cleaner?)

Reply:
I agree. I think because this function is called internally within the `stop` function, we can implement this function the way we want it (no callback) and handle the callback compatibility in the caller (`stop`).